src/snowflake/connector/constants.py

```diff
@@ -255,3 +255,6 @@ class IterUnit(Enum):
 
 S3_CHUNK_SIZE = 8388608  # boto3 default
 AZURE_CHUNK_SIZE = 4 * megabyte
+
+# TODO: all env variables definitions should be here
+ENV_VAR_PARTNER = "SNOWFLAKE_PARTNER"
```

Comment:
Same as what I said before @sfc-gh-madkins
```suggestion
ENV_VAR_PARTNER = "SF_PARTNER"
```